t/24-worker-webui-connection.t

```diff
@@ -206,6 +206,132 @@ subtest 'attempt to register and send a command' => sub {
     ) or diag explain \@happended_events;
 };
 
+my $default_ua = $client->ua;
```

Comment:
I suggest moving that into the subtest function so the scope is clear where the user agent is overriden